test/deploy/60_ocmagent.OcmAgent.yaml

```diff
@@ -4,10 +4,11 @@ metadata:
   name: ocmagent
   namespace: test-ocm-agent-operator
 spec:
-  ocmBaseUrl: "https://api.stage.openshift.com"
-  replicas: 1
-  services:
+  agentConfig:
+    ocmBaseUrl: "https://api.stage.openshift.com"
+    services:
   - service_log
```

Comment:
Missing an indent, this results in invalid yaml

Reply:
Thanks, fixed it!